CHANGELOG.md

```diff
@@ -3,6 +3,22 @@ Changelog
 
 Unreleased
 ----------
+### Added
+* Add support in generateComponentConfig for creating configuration from a terra repository that was installed as a package
```

Comment:
👍

Reply:
@bjankord I should clarify, at this time, we still need to add custom search patterns to generate this config, however folders like src, tests, components, etc will automatically be filtered out